packages/forklift-console-plugin/webpack.config.ts

```diff
@@ -86,6 +86,7 @@ const config: WebpackConfiguration & {
   },
   devServer: {
     static: './dist',
+    host: 'localhost',
```

Comment:
in some use cases you want to share your dev server with another developer, or run it on a VM and render on local browse

can you use some env var `process.env.DEV_SERVER_LISTEN_HOST` that defaults to `localhost`?

Reply:
I'll add it in a folowup